components/Events/List.js

```diff
@@ -55,15 +54,19 @@ const Overview = compose(
   graphql(query, {
     props: ({ data, ownProps: { router: { query: { slug } }, t } }) => {
       const error = data.error
+      const events =
+        data.events &&
+        data.events.map(event => ({ ...event, __parsedDate: parseDate(event.date) }))
       let event
-      if (slug && data.events && !error) {
-        event = data.events.find(event => event.slug === slug) || 404
+
+      if (slug && events && !error) {
+        event = events.find(event => event.slug === slug) || 404
       }
       return {
         loading: data.loading,
-        events: data.events,
-        error,
-        event
+        events,
```

Comment:
if `data.events` is falsy `events` should be an empty array

Reply:
If not mistaken,`data.events` can't be `null` or otherwise falsy due to schema restrictions?

```
type queries {
  [...]
  events: [Event!]!
  [...]
}
```

Reply:
ok, in that case the `data.events && …` test above is unnecessary

Reply:
True! Adopting.

Thank you.